test/templates/config.ru

```diff
@@ -0,0 +1,63 @@
+require_relative "fake_middleware"
+
+$current_dir = File.dirname(__FILE__)
+
+# node modules (e.g. "/node_modules/mocha/mocha.js")
+map "/node_modules" do
+  node_modules = File.expand_path($current_dir + "/../../node_modules")
+  unless Dir.exists?(node_modules)
+    raise "missing the node_modules directory"
```

Comment:
You can call `Kernel#abort` to do (almost) the same thing. Also `exit 1` is never called as noone is handling this exception. Ruby exits with a non-zero status code on exceptions anyway :)